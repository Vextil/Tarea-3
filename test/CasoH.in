Apilar texto1 200
Apilar texto2 100
Desapilar
CrearBinario texto100 100 1 texto050 100 texto150 200 texto250 100 FIN
FiltroBinario 1 120 > 2
RemoverBinario 2 texto250
BuscarABB 2 texto050
ImprimirBinario 2
Linealizar 1 1
CrearLista 2 texto1 texto100 texto2 texto3 texto4 FIN
ConstruirSet 1 1
ConstruirSet 2 2
CopiarSet 1 3
InsertarEnSet 1 2
RemoverDeSet 3 2
UnionInterDif 1 3 4 5 6 7
PerteneceASet 5 1
UnionInterDif 1 2 4 5 6 7
PerteneceASet 6 2
IngresarSolicitudes texto1 2 texto100 2 texto2 2 texto3 2 texto4 2 texto050 9 texto100 9 texto150 9 texto250 9 texto1 1 texto0 1 FIN
CancelarSolicitud
CancelarSolicitud
EnAlgunoSolicitudes 2 9 1
EnAmbosSolicitudes 2 9 2
SoloEnUnoSolicitudes 2 9 3
PerteneceASet 1 1
PerteneceASet 2 1
PerteneceASet 3 1
PerteneceASet 1 2
PerteneceASet 2 2
PerteneceASet 3 2
ListarSolicitudes
CorrerProcesos xc 100 notepad 300 FIN
PedirRecursos xc 5 FIN
SolicitanOMuyConsumidores 200 5
Fin
