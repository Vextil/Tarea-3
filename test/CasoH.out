1>OK
2>OK
3>(100,texto2)
4>OK
5>OK
6>OK
7>
 (100,texto050)
8>
 (100,texto100)
  (100,texto050)
9>OK
10>OK
11>OK
12>OK
13>OK
14>OK
15>OK
16>OK
17>SI NO SI SI 
18>OK
19>NO NO NO NO NO 
20>OK
21> 1
22> 1
23>OK
24>OK
25>OK
26>SI SI SI SI 
27>NO SI NO NO 
28>SI NO SI SI 
29>SI SI SI SI SI 
30>NO SI NO NO NO 
31>SI NO SI SI SI 
32>(2,texto1) (2,texto100) (2,texto2) (2,texto3) (2,texto4) (9,texto050) (9,texto100) (9,texto150) (9,texto250) 
33>OK
34>OK
35>notepad xc 
36>Fin
